api/migrations/0005_ginfocharacter.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.2 on 2017-07-03 09:09
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0004_characterhistory'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='GinfoCharacter',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('ginfo_marker_uid', models.TextField()),
+                ('character', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='api.Character')),
```

Comment:
I deleted and regenerated the migration, but `on_delete` was there again. I'll remove it by hand.